src/components/modals/ShareToTwitterModal.tsx

```diff
@@ -0,0 +1,50 @@
+import { Modal, Space } from 'antd'
+import { ProjectContext } from 'contexts/projectContext'
+import { useContext } from 'react'
+import { t, Trans } from '@lingui/macro'
+
+export default function ShareToTwitterModal({
+  visible,
+  onSuccess,
+  onCancel,
+}: {
+  visible?: boolean
+  onSuccess?: VoidFunction
+  onCancel?: VoidFunction
+}) {
+  const { metadata, handle } = useContext(ProjectContext)
+
+  async function tweet() {
+    const name = metadata?.twitter ? `@${metadata.twitter}` : metadata?.name
+    const twitterMsg = t`I just joined ${name}%2e See the live fundraiser%3a https%3a%2f%2fjuicebox.money%2f%23%2fp%2f${handle} %23juicebox`
```

Comment:
**suggestion**: to yield a nicer string for translation, this might be an option:
```suggestion
    const projectUrl = `https://juicebox.money/#/p/${handle}`
    const twitterMsg = encodeURIComponent(t`I just joined ${name}. See the live fundraiser: ${url}`)
```

Reply:
Nice!